Swiftification/Int.swift

```diff
@@ -25,10 +25,10 @@ import Foundation
 public extension Int {
     
     /// Random integer between min and max (inclusive)
-    static func random(range: Range<Int>) -> Int {
-        let offset = range.startIndex < 0 ? abs(range.startIndex) : 0
-        let min = UInt32(range.startIndex + offset)
-        let max = UInt32(range.endIndex + offset)
+    static func random(_ range: Range<Int>) -> Int {
```

Comment:
This only takes a `Range`. I think we can provide a version that takes a `ClosedRange`. The two range types can be converted into each other.